Compass129Sources/COMfile/C12COM9.ASC

```diff
@@ -926,39 +926,39 @@ m_all_save	push	hl
 	call	Mm_dos2_alloc
 	pop	hl
 	ret	c
-	LD	(IX+#00),B	;bewaar slotcode
+	LD	(IX+#00),B	;save slot code
 	INC	IX
-	LD	(IX+#00),A	;bewaar rambloknr
+	LD	(IX+#00),A	;save ram block number
 	INC	IX
 	dec	hl
 	ret
 Mm_dos2_alloc	LD	A,(stat_mem)
 	DEC	A
 	JR	NZ,m_Mm_alloc
-	LD	A,(primmap_slot)	;enkel dos 2
+	LD	A,(primmap_slot)	;only dos 2
 	OR	C	;try to allocate segments
 	LD	B,A
-	LD	A,#01	;allocate systemsegmenten
-	db	#c3	;Jump
+	LD	A,#01	;allocate system segments
+	db	#c3	;jump
 jp_all_seg	dw	0
 m_Mm_alloc	PUSH	IX
 	LD	DE,#4D0A
 	CALL	#FFCA
 	POP	IX
 	LD	A,H
 	OR	L
-	SCF		;hl=0000 =segmenten zijn op
+	SCF		;hl=0000 =segments are up
 	RET	Z
-	LD	B,L	;Memmansegmentcode blijkt te bestaan uit
-	LD	A,H	;H=mapperblok L=slotcode
+	LD	B,L	;Memman segment code appears to consist of
+	LD	A,H	;H=mapperblock L=lock code

```

Comment:
*slot* code rather than 'lock'.

Reply:
Ouch. _I knew_ that I was going to miss one of those. 😄